### ISSUE TITLE
BUG: CLI is not detecting cli flags set via env vars!!

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1982]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.13.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.13.1...4.13.2) (2021-08-05)

- build(deps-dev): bump @types/node from 16.4.10 to 16.4.12 [`#1980`](https://github.com/open-wa/wa-automate-nodejs/pull/1980)
- build(deps): bump puppeteer from 10.1.0 to 10.2.0 [`#1981`](https://github.com/open-wa/wa-automate-nodejs/pull/1981)
- Release 4.13.2 [`75e460c`](https://github.com/open-wa/wa-automate-nodejs/commit/75e460cc57ffddd3119cecf9205f55a5ae26f625)
- 🥚 patches update [`a671ac9`](https://github.com/open-wa/wa-automate-nodejs/commit/a671ac9c11647334aaf540b880f91172817d439c)
- 🐛 fix: CLI now detects flags via ENV VARS #1982 [`52d1a36`](https://github.com/open-wa/wa-automate-nodejs/commit/52d1a3647c43926f692cb87418a144782709e6b3)
- 🥚 patches update [`af98bcf`](https://github.com/open-wa/wa-automate-nodejs/commit/af98bcf949d1d6b3bf9f967a41e9f9f666bf288e)
